sematic/db/models/run.py

```diff
@@ -55,18 +57,18 @@ class Run(Base, JSONEncodableMixin, HasExternalJobsMixin):
         ID of the root run of the current graph. The root run corresponds to the
         entry point of the graph, i.e. the one corresponding to the future on which
         `resolve` was called.
-    description: Optional[str]
+    description : Optional[str]
         The run's description. Defaults to the function's docstring.
-    source_code: str
+    source_code : str
         The calculator's source code.
-    nested_future_id:
+    nested_future_id : Optional[str]
         If the run resulted in returning a new future, this contains the id of that
         future.
-    exception_metadata: Optional[ExceptionMetadata]
+    exception_metadata : Optional[ExceptionMetadata]
         The metadata for the exception from the calculator's execution, if any.
-    external_exception_metadata: Optional[ExceptionMetadata]
+    external_exception_metadata : Optional[ExceptionMetadata]
         The metadata for the exception from the external compute infrastructure, if any.
-    external_jobs_json:
```

Comment:
Would be nice to have these formatting rules applied by `black`. I need to do the same thing for code spacing.

Reply:
IMHO, the beautiful thing about black is that it is very limited in its configurability, which ends debates about code formatting. "Our standard for code formatting is whatever black does" saves sssooo much time talking about things like this in reviews and in team meetings. Everybody will be slightly disappointed by how the formatting works in some respect, but at least you save everybody the time trying to make the case for their personal pet peeve making it into the standard.

Reply:
You know it's a good compromise when everyone's disappointed.